### PR TITLE
updated sprockets 3.x to 4.x

### PR DESCRIPTION
**Pull request by pulkit-30**
#### Describe the changes you have made in this PR -
As @Kartikay304 mentioned, 

when we update sprockets 3.x to 4.x it throws double link errors
to solve double link error --> renamed the image files 
ex: `AsyncCounter_n.jpg` to `new_AsyncCounter_n.jpg`

after that it throws error saying 
`Can't resolve image into URL: undefined method start_with?' for /\.(?:svg|eot|woff|ttf)$/:Regexp`
to solve this , changed 
`Rails.application.config.assets.precompile << /\.(?:svg|eot|woff|woff2|ttf|otf)$/`  line to `Rails.application.config.assets.precompile << ["*.svg", "*.eot", "*.woff", "*.ttf"]` in config/initializers/assets.rb

after that, it throws an error for something stylesheet.css in assests>config>manifist.js
to solve that added `//= link application.css` line in app/assets/config/manifest.js

Now getting some warnings saying `DevTools failed to load source map`
Screenshot 2022-01-29 at 3 00 15 AM
Screenshot 2022-01-29 at 3 00 30 AM

Please check if these steps are correct for updating sprockets 3.x to 4.x or not and how to solve these errors??

### CODE REVIEW

**Comment by tachyons** [comment]
You can delete image files which are no longer needed

File: config/initializers/assets.rb [hunk 1]
```diff
@@ -14,4 +14,4 @@
 # Rails.application.config.assets.precompile += %w( admin.js admin.css )
 Rails.application.config.assets.precompile += %w[manifest.json webpush.js]
 Rails.application.config.assets.precompile += %w[simulator.css simulator.js application_sprockets.js application.js testbench.js testbench.css]
-Rails.application.config.assets.precompile << /\.(?:svg|eot|woff|woff2|ttf|otf)$/
+Rails.application.config.assets.precompile << ["*.svg", "*.eot", "*.woff", "*.ttf"]
```
Comment (tachyons) [inline config/initializers/assets.rb:17]: missing woff2 and otf

**Comment by tachyons** [comment]
Link for ref https://github.com/rails/sprockets/blob/master/UPGRADING.md

**Comment by pulkit-30** [comment]
Hey @tacyons,
I tried to solve these errors but i am unable


one similar issue is also mentioned [@rails community](https://github.com/rails/rails/issues/44260)

One way I found, is to redirect to the correct mapped file, but I don't think it is a correct way to handle this error

Screenshot 2022-02-03 at 10 04 27 PM

Please help me out with this

**Comment by tachyons** [comment]
Merging this as source map shouldn't be blocker for now

**Pull request merged**